src/v2/views/MonsterCollectionOverlay/index.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { MonsterCollectionContent } from ".";
+import { MonsterCollectionOverlayBase } from "./base";
+
+import "normalize.css";
+
+export default {
+  title: "MonsterCollection/MonsterCollectionContent",
+  component: MonsterCollectionContent,
+};
+
+export const FirstPage = () => (
+  <MonsterCollectionOverlayBase as="main">
```

Comment:
I explicitly opted out from OverlayBase because rendering and debugging dialog style on a Storybook felt quite awful. However, I think we should be able to use `MonsterCollectionOverlay` here since that component now manages the state too.